src/store/index.ts

```diff
@@ -140,12 +140,15 @@ export const store = createStore({
             context.commit("incrementBusyCounter");
             let page: Page<Identity>;
             if (await apiAvailable()) {
-                const chainStatusResponse = await getRequest<GetChainStatusResponse>(`/chains/status?wsProvider=${encodeURIComponent(wsAddress)}`);
-                if (chainStatusResponse.chainStatus.isIndexed) {
+                try {
                     const response = await getRequest<GetIdentitiesResponse>(`/identities/search?wsProvider=${encodeURIComponent(wsAddress)}&page=${currentPage}&limit=${this.state.identitySearchPagination.limit}&searchKey=${encodeURIComponent(searchData.searchTerm)}`);
                     page = response.identities;
-                } else {
-                    page = await searchIdentities(wsAddress, searchData.searchTerm, currentPage, this.state.identitySearchPagination.limit);
+                } catch (error) {
+                    if (["Provided node is not an archive node", "Chain is not indexed yet"].includes(error.message)) {
```

Comment:
You need to add the "." at the end of the messages and "Could not connect to endpoint." is missing

Reply:
I think we don't need to check "Could not connect to endpoint." error as Sascha has already implemented it. So if we are unable to connect to API the (apiAvailable) function will return false and we will fetch info with the npm package.